src/ui/toolbar/JoystickIndicator.qml

```diff
@@ -0,0 +1,102 @@
+/****************************************************************************
+ *
+ *   (c) 2009-2016 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+
+import QtQuick          2.5
+import QtQuick.Controls 1.2
+import QtQuick.Layouts  1.2
+
+import QGroundControl                       1.0
+import QGroundControl.Controls              1.0
+import QGroundControl.MultiVehicleManager   1.0
+import QGroundControl.ScreenTools           1.0
+import QGroundControl.Palette               1.0
+
+// Joystick Indicator
+Item {
+    width:          joystickRow.width * 1.1
+    anchors.top:    parent.top
+    anchors.bottom: parent.bottom
+    visible:        activeVehicle ? activeVehicle.sub : false
```

Comment:
Oh, wait. That's not possible. You don't have a vehicle at that point. Sorry about that.